imagevector/images.yaml

```diff
@@ -354,6 +354,10 @@ images:
 - name: alpine
   repository: europe-docker.pkg.dev/gardener-project/releases/3rd/alpine
   tag: "3.18.4"
+- name: alpine-conntrack
+  sourceRepository: github.com/gardener/alpine-conntrack
+  repository: europe-docker.pkg.dev/gardener-project/releases/gardener/alpine-conntrack
+  tag: "3.18.4"
```

Comment:
```suggestion
  tag: "3.19.0"
```

Reply:
I wanted to check the automatic pull request creation with the new version, but that only works after this is merged. Do you prefer to update directly to `3.19.0`?

Reply:
Ah, ok. Then let's first go with 3.18.4.